# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nss/hotfix-nosign.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nss/cmd/shlibsign/sign.sh.vanilla	2024-05-28 12:31:32.131590676 +0200
+++ nss/cmd/shlibsign/sign.sh	2024-05-28 12:31:42.380589688 +0200
@@ -10,6 +10,8 @@
 # 4: full path to DIST/OBJDIR/lib
 # 5: full path to library that is to be signed
 
+exit 0
+
 case "${3}" in
 WIN*)
     if echo "${PATH}" | grep -c \; >/dev/null; then
